docs/web-apps/automated-testing/cypress/yaml/v1.md

```diff
@@ -886,6 +886,31 @@ suites:
 
 ---
 
+### `shardGrepEnabled`
+<p><small>| OPTIONAL | BOOLEAN |</small></p>
+
+When sharding is configured, and used in conjunction with `@cypress/grep`, it may happen that some Spec files are executed without any tests matching the `@cypress/grep` filters.
```

Comment:
Just to avoid potential confusion around the word "executed", since the falsely picked up spec never _actually_ got executed. After all, it gets picked up by playwright and then skipped due to the grep.

```suggestion
When sharding is configured, and used in conjunction with `@cypress/grep`, it may happen that some spec files are allocated to VMs, just to be skipped by playwright in accordance with the `@cypress/grep` filters, thus wasting VM allocations.
```